lib/content-services/src/lib/node-comments/services/node-comments.service.spec.ts

```diff
@@ -81,5 +81,43 @@ describe('NodeCommentsService', () => {
                 responseText: JSON.stringify(fakeContentComments)
             });
         });
+
+        it('should cache the avatar URL when new comment is processed', () => {
+            const mockComment = {
+                id: '999',
+                content: 'test-message',
+                createdAt: '2024-07-14T12:00:00Z',
+                createdBy: {
+                    id: 'test-user',
+                    firstName: 'Test',
+                    lastName: 'User',
+                    email: 'test@user.com',
+                    avatarId: 'avatar123'
+                }
+            };
+
+            const contentService = (service as any).contentService;
+            spyOn(contentService, 'getContentUrl').and.returnValue('fake-url');
+
+            const result = (service as any).newCommentModel(mockComment);
+
+            expect(contentService.getContentUrl).toHaveBeenCalledWith('avatar123');
+            expect((service as any).avatarCache.get('test-user')).toBe('fake-url');
```

Comment:
similarly like in previous comments - we shouldn't refer to private fields by using any. Instead you can notice that avatarCache is used in public getUserImage function too so you can check if it stores correct data by checking if result of getUserImage function is correct